src/libstd/io/stdio.rs

```diff
@@ -18,6 +19,13 @@ use io::{self, BufReader, LineWriter};
 use sync::{Arc, Mutex, MutexGuard};
 use sys::stdio;
 
+/// Stdout used by print! and println! macroses
```

Comment:
Since this code is internal and docstring won’t appear in any public-facing documentation, I believe it is not worth updating this unless this bounces.